python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -881,12 +883,36 @@ def _assert_fast_slow_eq(left, right):
         assert_eq(left, right)
 
 
-class ProxyFallbackError(Exception):
+class FallbackError(Exception):
```

Comment:
For debugging purposes it might be helpful to store `func.__name__` from `_fast_slow_function_call` into the exception.